backend/api/Controllers/LookupController.cs

```diff
@@ -108,6 +108,62 @@ public IActionResult GetProjectRisks()
             return new JsonResult(risks.ToArray());
         }
 
+        /// <summary>
+        /// Get all of the purposes.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("project/purposes")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Models.LookupModel<int>>), 200)]
+        [SwaggerOperation(Tags = new[] { "lookup" })]
+        public IActionResult GetPurposes()
+        {
+            var purposes = _mapper.Map<Models.LookupModel<int>[]>(_pimsService.Lookup.GetPurposes());
+            return new JsonResult(purposes.ToArray());
+        }
+
+        /// <summary>
+        /// Get all of the moti classifications.
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet("project/moti/classifications")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(IEnumerable<Models.LookupModel<int>>), 200)]
+        [SwaggerOperation(Tags = new[] { "lookup" })]
+        public IActionResult GetMotiClassifications()
```

Comment:
We need a better naming convention for these classifications.  Are the original classifications relevant anymore?